src/main/scala/viper/gobra/translator/encodings/TypeEncoding.scala

```diff
@@ -192,23 +192,23 @@ trait TypeEncoding extends Generator {
 
   /**
     * Encodes statements.
-    * This includes make-statements.
+    * This includes new-statements and make-statements.
```

Comment:
I only see an implementation for new-stmts

Reply:
This is a vestige of a previous implementation, I forgot to change that